aio/aio-proxy/aio_proxy/doc/open-api.yml

```diff
@@ -0,0 +1,381 @@
+info:
+  title: API Recherche d’entreprises
+  version: 1.0.0
+  contact:
+    email: recherche-entreprises@api.gouv.fr
+    name: Contact API Recherche d’entreprises
+    url: https://recherche-entreprises.api.gouv.fr/
```

Comment:
We need to add a redirect https://recherche-entreprises.api.gouv.fr/ -> https://api.gouv.fr/api-recherche-entreprises